multiraft/multiraft.go

```diff
@@ -765,6 +789,23 @@ func (s *state) propose(p *proposal) {
 	p.fn()
 }
 
+func (s *state) coincideRaftLeader(groupID proto.RangeID) error {
+	if _, ok := s.groups[groupID]; !ok {
+		return nil
+	}
+	if log.V(3) {
+		log.Infof("node %v coincideRaftLeader group %v", s.nodeID, groupID)
+	}
+	status := s.Status(groupID)
+	if log.V(3) {
```

Comment:
Both of these log statements will happen in all cases. Did you mean for the second log message to occur after the `if` statement, so it would only log if the Campaign is actually kicked off?